tests/tests_pytorch/utilities/test_remote_filesystem.py

```diff
@@ -1,106 +0,0 @@
-# Copyright The PyTorch Lightning team.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-import os
-
-import fsspec
-import pytest
-
-from pytorch_lightning import Trainer
-from pytorch_lightning.callbacks import ModelCheckpoint
-from pytorch_lightning.demos.boring_classes import BoringModel
-from pytorch_lightning.loggers import TensorBoardLogger
-
-GCS_BUCKET_PATH = os.getenv("GCS_BUCKET_PATH", None)
```

Comment:
have we considered adding the bucket path to CI and moving the requirement to test.txt?

Reply:
I mentioned that as a different option in https://github.com/Lightning-AI/lightning/pull/14788#discussion_r984104347.

I lean towards removal because we have no logic specific to gcs in the codebase. This is all encapsulated by `fsspec`, and we have tests for our integration with it. I assume fsspec has gcs specific tests.

Reply:
Totally fine with me, just making sure we're not removing something that was needed